connector/datadogconnector/connector_native.go

```diff
@@ -158,3 +176,20 @@ func (c *traceToMetricConnectorNative) run() {
 		}
 	}
 }
+
+// fork of https://github.com/DataDog/datadog-agent/blob/7642cf1aa659f82744038602044a8a00aa1a0dfb/pkg/trace/agent/obfuscate.go#L109
```

Comment:
is this sustainable long-term? Is it possible we could add this as a function to `pkg/obfuscate` instead so that it doesn't get forgotten when obfuscate folks make changes to their obfuscateStatsGroup calculations?

Reply:
we are already having to import pkg/obfuscate; I feel like we could add a function there that takes the `c.obfuscator` as a parameter instead

Reply:
submitted a PR to modify the signature of this function in trace agent: https://github.com/DataDog/datadog-agent/pull/32064